lua/lib/lattice.lua

```diff
@@ -103,7 +103,7 @@ function Lattice:pulse()
         pattern.phase = pattern.phase + 1
         local swing_val = (2*pattern.swing/100)
         if not pattern.downbeat then 
-          swing_val = (2*(100-pattern.swing)/100)
+          swing_val = 1
```

Comment:
@sixolet one is the loneliest number. 🎶 thank you for spending some cycles analyzing and looking at this!!

@schollz what do you think think? i never understood this "why" behind this math. if this makes swing behave more intuitively i am all for it.

Reply:
yeah this is way more intuitive! thanks @sixolet